src/collections/_documentation/workflow/integrations/global-integrations.md

```diff
@@ -957,9 +957,51 @@ In the next section, we'll walk you through configuring your notification settin
 
     [{% asset slack-alert-message.png %}]({% asset slack-alert-message.png @path %})
     
-#### Deleting the legacy Slack integration
+#### Deleting the legacy project-based Slack integration
 
-Once you configure the global Slack integration and Alert Rules, you can disable the old Slack integration. You’ll need to go to each project that has it enabled and disable it. We recommend disabling the legacy integration after setting up the global integration.
+Once you configure the global Slack integration and Alert Rules, you can disable the old project-based Slack integration. You’ll need to go to each project that has it enabled and disable it. We recommend disabling the legacy project-based integration after setting up the global integration. If you're looking to upgrade your global integration, please refer to the information below under **Upgrading Slack**.
+
+### Upgrading Slack
+
+#### How do I know if I need to upgrade? 
+
+If you installed the global Slack integration on a Sentry organization prior to May 8, 2020, you'll need to upgrade your integration. If you're unsure whether or not that includes your organization, please visit your integrations page via the Organization Settings > Integrations. You will see a call to action for the Slack integration. (screenshot?) (Please note we are still rolling this out, if you don't see this yet you may be in a later wave)
+
```

Comment:
I'm not sure the best way to phase this but basically we want to have these docs available to link in the upgrade flow incase people have more questions. However, at first we will only be rolling it out to about ~5% of users that need to upgrade. We should have it available for all users (that have an outdated slack app) relatively soon but the idea was to minimize the impact incase people have a lots of questions/getting really confused etc. If we start with only 5% we can update wording/fix whats not clear for people going forward.